applications/serial_lte_modem/doc/NRFCLOUD_AT_commands.rst

```diff
@@ -183,200 +183,89 @@ Example
-* The ``<uncertainty>`` value represents the certainty of the result.
-
-Example
-~~~~~~~
-
-::
-
-  AT%XSYSTEMMODE=1,0,0,0
-
-  OK
-  AT+CFUN=1
-
-  OK
-  AT#XNRFCLOUD=1
-
-  OK
-  #XNRFCLOUD: 1,0
-  AT#XCELLPOS=1
-
-  OK
-
-  #XCELLPOS: 0,35.455833,139.626111,1094
-
-  AT%NCELLMEAS
-
-  OK
-
-  %NCELLMEAS: 0,"0199F10A","44020","107E",65535,3750,5,49,27,107504,3750,251,33,4,0,475,107,26,14,25,475,58,26,17,25,475,277,24,9,25,475,51,18,1,25
+* ``0`` - Do not include cellular network information in the location request.
+* ``1`` - Use single-cell cellular network information (only the serving cell).
+* ``2`` - Use multi-cell cellular network information (the serving and possibly neighboring cells).
+  To use ``2``, you must first issue the ``AT%NCELLMEAS`` command and wait for its result notification.
+  The cellular network information included in the location request will be the one received from ``AT%NCELLMEAS``.
+  Thus, it should be run as little time before sending the location request as possible if the goal is to have the most up-to-date location information.
+  Furthermore, whenever sending a location request in single-cell mode the previously saved multi-cell cellular network information is flushed.
 
-  AT#XCELLPOS=2
-
-  OK
-
-  #XCELLPOS: 1,35.534999,139.722362,1801
-  AT#XCELLPOS=0
-
-  OK
-
-Read command
-------------
-
-The read command allows you to check the cellular location service status.
-
-Syntax
-~~~~~~
-
-::
-
-   #XCELLPOS?
-
-Response syntax
-~~~~~~~~~~~~~~~
-
-::
-
-   #XCELLPOS: <cellpos_status>
-
-* The ``<cellpos_status>`` value is an integer.
-  When it returns the value of ``1``, it means that the cellular location service is started.
-
-Example
-~~~~~~~
-
-::
-
-  AT#XCELLPOS?
-
-  #XCELLPOS: 1
-
-  OK
-
-Test command
-------------
-
-The test command tests the existence of the command and provides information about the type of its subparameters.
+The ``<wifi_pos>`` parameter can have the following integer values:
 
-Syntax
-~~~~~~
+* ``0`` - Do not include Wi-Fi access point information in the location request.
+* ``1`` - Use Wi-Fi access points information.
+  The access points must be given as additional parameters to the command.
+  At least 2 (:c:macro:`NRF_CLOUD_LOCATION_WIFI_AP_CNT_MIN`) must be provided.
+  The maximum number of access points is limited by the AT command parameter count limit, :ref:`CONFIG_SLM_AT_MAX_PARAM <CONFIG_SLM_AT_MAX_PARAM>`.
 
-::
+The ``<MAC x>`` parameter is a string.
+It indicates the MAC address of a Wi-Fi access point and must be formatted as ``%02x:%02x:%02x:%02x:%02x:%02x`` (:c:macro:`WIFI_MAC_ADDR_TEMPLATE`).
 
-   #XCELLPOS=?
+The ``<RSSI x>`` parameter is an optional integer.
+It indicates the signal strength of a Wi-Fi access point in dBm, between ``-128`` and ``0``.
+If provided, it must follow the MAC address parameter of the access point.
+Providing the RSSI parameters helps improve the accuracy of the Wi-Fi location.
```

Comment:
My idea was that if I would try this command, it would be faster for me to remember/see the SSID:s of the WLAN's that are currently in the area. This might be useful for occasional use, but has problems with WLAN:s which have same SSID:s.

However, now that I look at the original command, it needed both ssid and mac, so it is OK to remove SSID.